sdk/synapse/azure-synapse-artifacts/azure/synapse/artifacts/models/_artifacts_client_enums.py

```diff
@@ -127,6 +136,18 @@ class DatasetReferenceType(with_metaclass(_CaseInsensitiveEnumMeta, str, Enum)):
     DATASET_REFERENCE = "DatasetReference"
 
 class DayOfWeek(with_metaclass(_CaseInsensitiveEnumMeta, str, Enum)):
+    """The days of the week.
+    """
+
+    SUNDAY = "Sunday"
+    MONDAY = "Monday"
+    TUESDAY = "Tuesday"
+    WEDNESDAY = "Wednesday"
+    THURSDAY = "Thursday"
+    FRIDAY = "Friday"
+    SATURDAY = "Saturday"
+
+class DaysOfWeek(with_metaclass(_CaseInsensitiveEnumMeta, str, Enum)):
```

Comment:
curious - what's the difference between this and DayOfWeek above

Reply:
Good catch!